src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/widget/WidgetPageObject.java

```diff
@@ -77,7 +77,9 @@ public WidgetPageObject createIncorrect(String articleName) {
    * Make sure that there is a widget of its type loaded on the page.
    */
   public boolean isLoaded() {
-    boolean result = isWidgetVisible(0);
+//    boolean result = isWidgetVisible(0);
+    boolean result = false;
+//    logVisibility(result);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3ACommentedOutCodeLine)